packages/tinacms/src/components/media/media-item.tsx

```diff
@@ -61,6 +55,17 @@ export function MediaItem({
   )
 }
 
+function renderPreview(item: Media) {
+  if (item.previewSrc) {
+    if (item.type === 'image' || item.type === 'file') {
+      return <img src={item.previewSrc} alt={item.filename} />
+    } else if (item.type === 'video') {
+      return <video src={item.previewSrc} />
```

Comment:
I'm wondering if its a good idea to try and use the actual videos as the preview src. We may just want to use an icon.